py/picca/io.py

```diff
@@ -43,6 +43,34 @@ def read_dlas(fdla):
 
     return dlas
 
+def read_absorbers(file_absorbers):
```

Comment:
why not give it a fits file? It is so much better than ascii...

Reply:
Analogous to the DLA catalog and line mask.